CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Unique all available pre-release versions when displaying  
+  [Samuel Giddins](https://github.com/segiddins)
```

Comment:
@segiddins I added your name here since you did this change locally, I am only upstreaming it.